src/Library/demos/Scale/main.js

```diff
@@ -0,0 +1,33 @@
+const scale_one = workbench.builder.get_object("one");
+const scale_two = workbench.builder.get_object("two");
+const scale_disabled = workbench.builder.get_object("disabled");
+
+scale_two.add_mark(25.0, "left", "A");
+scale_two.add_mark(50.0, "right", "B");
+scale_two.add_mark(75.0, "right", "C");
+scale_two.add_mark(100.0, "right", "D");
```

Comment:
ha yes - Blueprint doesn't support marks unfortunaly so we have to do it in JS

https://gitlab.gnome.org/jwestman/blueprint-compiler/-/issues/62

Reply:
It's already in JS. So shall i leave it unchanged?

Reply:
yes - it's good as is